dashboard/src/components/AppView/AppControls.tsx

```diff
@@ -103,6 +103,39 @@ class AppControls extends React.Component<IAppControlsProps, IAppControlsState>
   private togglePurge = () => {
     this.setState({ purge: !this.state.purge });
   };
+
+  private renderUpgradeButton = () => {
+    const { app, latest } = this.props;
+    const deleted = app.info && app.info.deleted;
+    const isOutdated =
+      app.chart &&
+      app.chart.metadata &&
+      app.chart.metadata.version &&
+      latest &&
+      app.chart.metadata.version !== latest;
+    let upgradeButton = null;
+    // If the app has been deleted hide the upgrade button
+    if (!deleted) {
+      upgradeButton = (
+        <button className="button" onClick={this.handleUpgradeClick}>
+          Upgrade
+        </button>
+      );
+      // If the app is outdated highligh the upgrade button
+      if (isOutdated) {
+        upgradeButton = (
+          <div className="tooltip">
+            <button className="button upgrade-button" onClick={this.handleUpgradeClick}>
+              <span className="upgrade-text">Upgrade</span>
+              <ArrowUpCircle color="white" size={25} fill="#20cb15" className="notification" />
```

Comment:
is #20cb15 from the Hex colour palette?

Reply:
We should aim to use colours from the Hex design palette: https://design.bitnami.com/category/Foundations/Variables/#Colors